RELEASING.md

```diff
@@ -0,0 +1,18 @@
+# Release process notes
+
+## Prerelease
+
+- create a new branch with some name
+- write up changelog changes (manually), commit that
+- yarn lerna version --no-push 6.2.0-alpha.2
```

Comment:
Perhaps we can drop some notes as subpoints where required which explains what the cmd does and what we'd expect to see, eg 
- yarn lerna version --no-push 6.2.0-alpha.2
  - `yarn lerna version` bumps the version of all packages changed since the last release. 
  - Adding `--no-push` means the changes aren't pushed to git, we do this because.... 
  - We manually specify the tag at the moment because we're doing alpha releases but down the track we can use keywords like `minor` or `patch`.